app/controllers/wikis_controller.rb

```diff
@@ -93,6 +95,13 @@ def destroy
   def downgrade
     current_user.role = :standard
     current_user.save
+    @wikis = Wiki.all
+    @wikis.each do |wiki|
+      if current_user == wiki.user && wiki.private?
+        wiki.private = 'false'
+        wiki.save
+      end
+    end
```

Comment:
Lines 98-104 can take advantage of `User`-`has_many`-`:wikis` association and `where(conditions)` to be simplified to:

```diff
-    @wikis = Wiki.all
-    @wikis.each do |wiki|
-      if current_user == wiki.user && wiki.private?
-        wiki.private = 'false'
-        wiki.save
-      end
-    end
+    wikis = current_user.wikis.where(private: true)
+    wikis.each { |wiki| wiki.update(private: false) }
```